feature/amazon/src/main/java/com/revenuecat/purchases/amazon/DefaultPurchasingServiceProvider.kt

```diff
@@ -1,11 +1,15 @@
 package com.revenuecat.purchases.amazon
 
+import android.app.Activity
 import android.content.Context
+import android.content.Intent
+import android.os.ResultReceiver
 import com.amazon.device.iap.PurchasingListener
 import com.amazon.device.iap.PurchasingService
 import com.amazon.device.iap.model.FulfillmentResult
 import com.amazon.device.iap.model.RequestId
 
+const val PROXY_AMAZON_BILLING_ACTIVITY_REQUEST_CODE = 192342
```

Comment:
what in the world

Reply:
how did you determine this number?

Reply:
That's basically a "tag" that gets set when starting an activity and allows to determine the result corresponds for this specific code, so it doesn't really matter what value it has. However, I believe it has to be a number less than `65535`, so I actually think this number is not valid. That said, if we go with the `BroadcastReceiver` idea, we won't need to pass this value, so it can be removed then